### PR TITLE
fix: do not show the minimized grid if a call is already ongoing

### PR DESCRIPTION
**Pull request by atomrc**

### CODE REVIEW

File: app/script/components/list/conversationListCallingCell.js [hunk 1]
```diff
@@ -143,7 +143,15 @@ z.components.ConversationListCallingCell = class ConversationListCallingCell {
     this.showParticipants = ko.observable(false);
 
     this.showVideoPreview = ko.pureComputed(() => {
-      return this.isVideoCall() && (this.multitasking.isMinimized() || !this.callIsConnected());
+      const nbOfOtherOngoingCalls = this.calls()
```
Comment (gregor) [inline app/script/components/list/conversationListCallingCell.js]: We do not care about the number. You can just turn this into a boolean. Much easier on the naming as you can go with `hasOtherOngoingCall`

Comment (atomrc) [reply app/script/components/list/conversationListCallingCell.js]: Fair point :+1:

File: app/script/components/list/conversationListCallingCell.js [hunk 2]
```diff
@@ -143,7 +143,14 @@ z.components.ConversationListCallingCell = class ConversationListCallingCell {
...
     this.showVideoPreview = ko.pureComputed(() => {
-      return this.isVideoCall() && (this.multitasking.isMinimized() || !this.callIsConnected());
+      const hasOtherOngoingCalls =
+        this.calls()
+          .filter(call => call.id !== this.call().id)
+          .filter(call => call.state() === z.calling.enum.CALL_STATE.ONGOING).length > 0;
+
+      return (
+        !hasOtherOngoingCalls && this.isVideoCall() && (this.multitasking.isMinimized() || !this.callIsConnected())
+      );
```
Comment (gregor) [inline app/script/components/list/conversationListCallingCell.js]: May I ask you to pull the last `or` bit out into a separats variable and use that in the return. I suppose the return fits into a single line then.

Comment (atomrc) [reply app/script/components/list/conversationListCallingCell.js]: There you go :)

**Pull request merged**